microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -103,24 +103,20 @@ At this point you should have WebView2 Runtime installed, built a WebView2 proje
     | `is_legacy` | Set to `false`, which tells Selenium that you are driving the new Chromium-based Microsoft Edge browser. |  
     | `"webview2"` | A string that tells Selenium you are driving WebView2. |  
     
-1.  Next, set `edgeOptions.BinaryLocation` to the file path of your WebView2 project runtime, create a string named `msedgedriverDir` that provides the file path to where you installed [Microsoft Edge Driver][MicrosoftDeveloperMicrosoftEdgeWebDriverDownloads], and create a string named `msedgedriverExe` to store the name of the Microsoft Edge Driver runtime.  By default, the runtime is named `msedgedriver.exe`. Use these two strings to construct the `EdgeDriverService` object as shown below.  Finally, create the `EdgeDriver` object using `EdgeDriverService` and `EdgeOptions`.  
+1.  Next, configure `edgeOptions` to use Chromium and WebView by setting the member variables `UseChromium` and `UseWebView` to true. Then, set `edgeOptions.BinaryLocation` to the file path of your WebView2 project runtime. Finally, create the `EdgeDriver` object using `EdgeOptions`.  
```

Comment:
Is "WebView2 project runtime" the proper terminology for an app that hosts WebView2? This might be confused with the WebView2 runtime itself (e.g. msedgewebview2.exe). We also don't refer to Edge Driver as a "runtime" in other parts of the documentation. I'd prefer to just refer to it as "Edge Driver", or remove it from this sentence because the code snippet doesn't actually specify a path to Edge Driver.

Reply:
Just to confirm, are you suggesting changing '...WebView2 project runtime" --> "...Edge Driver"?

Reply:
@bwalderman

Reply:
@jasonstephen15 , `edgeOptions.BinaryLocation` points to the user's WebView2 project, not an Edge Driver binary. I'd recommend a term like "WebView2 application binary", "WebView2 project binary", or just "WebView2 project" here. I'd also avoid using the terms "runtime" or "driver" here since these can easily be confused with the WebView2 runtime (msedgewebview2.exe) or Edge Driver (msedgedriver.exe) which are separate processes.

Reply:
We typically call this the "host app", so "file path of your host app binary." would be clear and accurate I think.